types/bootstrap-fileinput/index.d.ts

```diff
@@ -119,6 +120,44 @@ declare namespace BootstrapFileInput {
          * Theming
          */
         theme?: string;
+        /**
+         * Whether file selection is mandatory before upload (for ajax) or submit of the form (for non-ajax). When set to true, and if files are not
+         * selected before upload, this will show the error message as set in msgFilerequired. Defaults to false.
+         */
+        required?: boolean;
```

Comment:
can I propose, optional change (here and below)
```ts
* @default false
```

Reply:
Sounds good! I'll add those.

Reply:
@peterblazejewicz I've added `@default` values for the booleans you mentioned and replaced the comments `Defaults to false` and `Defaults to true` with `@default false` and `@default true` where applicable. Also added `@default`s for the booleans in `FileActionSettings` based on the [docs](https://plugins.krajee.com/file-input/plugin-options#fileActionSettings).